### DISCUSSION TITLE
how to create Parameter File

### DISCUSSION THREAD

**Discussion by smallriver666** [SeisSol/SeisSol#556] [Q&A] [answered]
Dear all, I have some problems while creating parameter file,
![image](https://user-images.githubusercontent.com/104721358/169679079-63040416-30ac-4b5d-8238-fd7e740680b2.png)
I have done step 1 and step 2 according to the Configuration file, but I am not clear how step 3.Create your Parameter File is implemented.
What can I do to generate this Commented parameter file?
![image](https://user-images.githubusercontent.com/104721358/169679246-41e8fc47-f676-429a-a572-7061b1fa185a.png)
Also where can I find all the input files mentioned in step 4, and where can I find *.yaml files for setting model parameters
I would appreciate it if you can help me.

**Comment by smallriver666** [comment]
![image](https://user-images.githubusercontent.com/104721358/169694177-3aab3338-6145-4aba-9e22-bfc80fdb7912.png)
![image](https://user-images.githubusercontent.com/104721358/169694262-a5648977-019e-4c43-b17b-78182e5af67d.png)
Is it modified on the basis of this file, or I create a new file through `vim parameter-file`, and which of the following mesh files are mentioned here?

**Comment by smallriver666** [comment]
![image](https://user-images.githubusercontent.com/104721358/169696142-6e2c793e-9032-4e50-bc30-5b27e1518533.png)
maybe this file

**Comment by sebwolf-de** [comment, marked as answer]
You can copy the parameter file from our documentation and then edit it with any text editor to fit your needs. You have to generate the `yaml` files by yourself. Again, these are plain text files, which you can edit with any editor.
You should start with one of our examples: https://github.com/SeisSol/Examples. LOH1 is very easy, TPV5 shouldn't be much harder. There you will see which files you need.

**Comment by smallriver666** [comment]
I appreciate you taking the time to answer me, but I still want to ask if this `yaml` file can be automatically generated by the computer, or should I write it myself. In addition, if I just run your example, can this file directly copy the `yaml` file generated in your example library.
I'm just starting to learn the software and may have asked some very simple questions.

**Comment by sebwolf-de** [comment]
No, it can not be automatically generated by the computer. Through the yaml file, you tell the computer, which material parameters you want to use in your simulation. You can copy all files around as you wish. Actually, for a beginner, I would recommend running LOH1 or TPV5 first and then step by step adjust the setup to your needs.

**Reply by smallriver666** [reply]
> No, it can not be automatically generated by the computer. Through the yaml file, you tell the computer, which material parameters you want to use in your simulation. You can copy all files around as you wish. Actually, for a beginner, I would recommend running LOH1 or TPV5 first and then step by step adjust the setup to your needs.

thank you very much, I will have a try, and  I should learn some theoretical knowledge first, otherwise I can't understand a lot of code
